usecases/MinimumWorkingExample/upload_scripts/gen_json_from_metadata_yaml.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
Don't submit that file. This was more for us and internal discussions

Reply:
done